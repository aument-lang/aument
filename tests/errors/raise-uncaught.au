func hello() {
    raise "Hello World\n";
}

hello();