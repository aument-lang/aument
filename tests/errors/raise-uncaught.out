interpreter error(12) in -: uncaught exception
2 |     raise "Hello World\n";
interpreter error(7) in -: came from here
5 | hello();
